perun-openapi/openapi.yml

```diff
@@ -1630,6 +1803,8 @@ tags:
     description: VosManager RPC API in Perun
     externalDocs:
       url: https://perun-aai.org/documentation/technical-documentation/rpc-api/rpc-javadoc-VosManager.html
+  - name: TasksManager
+    description: TasksManager RPC API in Perun
 
```

Comment:
You can add here a link to the external docs. https://perun-aai.org/documentation/technical-documentation/rpc-api/rpc-javadoc-TasksManager.html